### DISCUSSION TITLE
About redshift drift

### DISCUSSION THREAD

**Discussion by VitalyPetrikov** [a-cosmology-group/acg#166] [Cosmology Models]
Logically, if z is an indicator of the speed of the galaxy, then the change towards increasing z (drift) is an indicator of the positive acceleration of the galaxy. And, therefore, this is not an indicator of the expansion of the universe (space). Because if the drift reduces the shift of the spectrum, then there should be a compression of the universe (space) in individual neighborhoods of this galaxy?
And negative drift is quite possible for galaxies that have stopped accelerating and, therefore, are very weakly inhibited by the general attraction to the center of space (Section 4 of the Appendix in attached file). Naturally, in this case, longer observation of the spectrum shift is necessary.
[VVPetrikov About recession of galaxies 2nd edition 10-2023.pdf](https://github.com/a-cosmology-group/acg/files/13175039/VVPetrikov.About.recession.of.galaxies.2nd.edition.10-2023.pdf)

**Comment by RedshiftDrift** [comment]
>  then the change towards increasing z (drift) is an indicator of the positive acceleration of the galaxy

No, redshift-drift is present in a non-accelerating universe because of expansion as a function of time, this is not related to acceleration described by $\Lambda$.  The function is nearly exponential, so there is no "compression of space" as you describe but instead just the expected compression of space in the past given by $a(t)$.

**Reply by bgreenman** [reply]
redshiftdrift is right

**Comment by VitalyPetrikov** [comment]
Louis, obviously I didn't express myself accurately. I'll try to improve.
    My work reveals that galaxies fly apart under the influence of gravity. And that the speed of galaxies increases from almost zero to almost light speed in motionless space. Gravity is a force and, accordingly, galaxies receive acceleration from it. And therefore the redshift drift is an expression of the acceleration of distant galaxies. But when the acceleration of a galaxy ends, it switches to the mode of motion by inertia - the Hubble parameter zone. And in a situation of movement by inertia, there is no redshift drift. The difference between the universe considered in the essay is that in it there is a sharp boundary between the location of galaxies moving at an accelerated rate and galaxies moving by inertia. According to my assumptions (if the age of the universe is 40 billion years), the position of this boundary is at a distance of 13.3 billion light years (Fig. 3 in the attached file).
    But in a universe with an asymptotic expansion of space, such a boundary does not exist in principle. And here I ask the question, is redshift drift present in all galaxies? Is drift more noticeable in distant galaxies than in nearby galaxies?
Is there a boundary separating galaxies with and without redshift drift?
[VVPetrikov About recession of galaxies 2nd edition 10-2023.pdf](https://github.com/a-cosmology-group/acg/files/13175234/VVPetrikov.About.recession.of.galaxies.2nd.edition.10-2023.pdf)

**Reply by bgreenman** [reply]
We could only tell the speed of distant galaxies from our solar system-Milky way vantage point

**Comment by VitalyPetrikov** [comment]
I completely agree with this, but what is the point of this remark

**Reply by bgreenman** [reply]
The point is we humans think at this point in the 21st century we are close to solving all our cosmological paradoxes ,but we are just beginning to have the technology to start looking for solutions: its a never ending quest
